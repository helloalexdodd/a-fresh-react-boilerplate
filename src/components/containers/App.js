import React from 'react';
import { Container, P } from './App.style';
import logo from '../../assets/logo.png';

const App = () => (
	<Container>
		<img src={logo} alt='A Fresh React Boilerplate' />
		<P>Thanks for downloading!</P>
	</Container>
);

export default App;
