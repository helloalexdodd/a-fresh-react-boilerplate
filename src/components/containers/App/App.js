import React from 'react';
import Logo from 'assets/logo.png';
import { Container, P } from './style/App.style';

const App = () => (
  <Container>
    <img src={Logo} alt="A Fresh React Boilerplate" />
    <P>Thanks for downloading!</P>
  </Container>
);

export default App;
