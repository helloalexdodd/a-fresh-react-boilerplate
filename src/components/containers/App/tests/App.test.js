import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { axe } from 'jest-axe';
import App from '../App';

afterEach(cleanup);

describe('App', () => {
  it('should take a snapshot', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});

it('should have no accessibility errors', async () => {
  const { container } = render(<App />);
  const results = await axe(container);
  expect(results).toHaveNoViolations();
});
