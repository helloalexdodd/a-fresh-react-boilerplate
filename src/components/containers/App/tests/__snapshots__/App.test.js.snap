// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should take a snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #f2f2f2;
  height: 100vh;
  color: #3a454f;
}

.c1 {
  font-size: 2rem;
}

<div>
  <div
    class="c0"
  >
    <img
      alt="A Fresh React Boilerplate"
      src="test-file-stub"
    />
    <p
      class="c1"
    >
      Thanks for downloading!
    </p>
  </div>
</div>
`;
